requirements.txt

```diff
@@ -1,17 +1,4 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-PyYAML==3.10
-UnittestZero
-certifi==0.0.8
-chardet==1.0.1
-py==1.4.9
-pytest==2.2.4
-pytest-mozwebqa==1.1
-pytest-xdist==1.8
-requests==1.2.0

```

Comment:
This is only required via pytest-mozwebqa, so let's leave that to manage the dependency.